tox.ini

```diff
@@ -12,9 +12,9 @@ deps =
     flake8-string-format
 
 commands =
-   pytest tests --cov argoverse --cov-append --cov-branch --cov-report=term-missing
-   flake8 --max-line-length 120 --ignore E203,E704,E711,E722,E741,W291,W293,W391,W503,F821,F401,F811,F841,P101,G004,G002,I201,I100,I101 --enable-extensions G argoverse
-   mypy --ignore-missing --strict argoverse
+   pytest tests
```

Comment:
have we verified that these options get picked up from the pyproject.toml file in practice?

Reply:
Some of these can be verified by looking at the `tox` run in our Github workflow. I'll verify once again after we've settled on the other changes.